snap/implicit.go

```diff
@@ -56,6 +56,7 @@ var implicitClassicSlots = []string{
 	"modem-manager",
 	"optical-drive",
 	"camera",
+	"kernel-module",
```

Comment:
This should be in implicitSlots since it should be available to more than just classic systems, AIUI.